app/assets/stylesheets/components/navigation.scss

```diff
@@ -2,9 +2,11 @@
 
 .crayons-link {
   color: var(--link-color);
+  text-decoration: none;
```

Comment:
@lisasy this is the change I'm referring to.

Reply:
@Ridhwana The underline isn't appearing for other places using `.crayons-link`, so I am going to ask @ludwiczakpawel to review this and see why the underline is appearing for these navigation items, when that isn't the case in `/readinglist` or `/notifications`, for example.

My current hypothesis is that `text-decoration: underline` is currently rendering for any `a` item in `/admin` because `/admin` is still pulling in styles from Bootstrap. Eventually, we will deprecate Bootstrap completely so that all UI stylings will be crayons-only. 

If this is the case, then I would recommend using an admin-specific class name with `text-decoration: none` for these `a` elements. When we fully deprecate Bootstrap from Admin Portal, we can also remove these admin-specific class names.

Reply:
What @lisasy says is correct - it's coming from bootstrap and it's not an issue on user facing side of DEV because we use bootstrap only in admin section. And as long as we use it, we will likely face some bigger or smaller styling conflicts like this one.

Until we get rid of bootstrap, I highly recommend making "one-off" changes and fixes like that in admin specific stylesheet file: https://github.com/forem/forem/blob/master/app/assets/stylesheets/admin.scss - where you could also leave a comment or something explaining WHY this fix is there so once we get to refactoring this, it will be a bit easier :).

Reply:
Sounds good - Thanks you both  😊